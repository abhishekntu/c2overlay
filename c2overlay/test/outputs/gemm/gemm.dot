digraph graphname {
N55 [color="black", label="N79N98_mul_Imm_6_N79_add_N98_N55"];
N74 [color="black", label="store_Imm_5_N106_N74"];
N78 [color="black", label="store_Imm_1_N102_N78"];
N70 [color="black", label="N53N94_mul_N53_add_N94_N70"];
N54 [color="black", label="mul_Imm_11_N73_N54"];
N73 [color="black", label="store_Imm_3_N107_N73"];
N69 [color="black", label="mul_N52_N69"];
N79 [color="black", label="store_Imm_7_N101_N79"];
N58 [color="black", label="mul_Imm_11_N41_N58"];
N68 [color="black", label="mul_Imm_11_N54_N68"];
N80 [color="black", label="store_Imm_2_N100_N80"];
N49 [color="black", label="N74N89_mul_Imm_6_N74_add_N89_N49"];
N48 [color="black", label="mul_Imm_11_N60_N48"];
N65 [color="black", label="mul_N46_N65"];
N64 [color="black", label="N47N75_mul_N47_add_N75_N64"];
N63 [color="black", label="mul_Imm_11_N44_N63"];
N62 [color="black", label="mul_Imm_11_N45_N62"];
N61 [color="black", label="mul_N42_N61"];
N60 [color="black", label="N43N65_mul_N43_add_N65_N60"];
N41 [color="black", label="mul_Imm_11_N91_N41"];
N40 [color="black", label="N93N68_mul_N93_add_N68_N40"];
N43 [color="black", label="N95N58_mul_Imm_6_N95_add_N58_N43"];
N42 [color="black", label="mul_Imm_11_N96_N42"];
N45 [color="black", label="N63N81_mul_N63_add_N81_N45"];
N44 [color="black", label="N66N55_mul_N66_add_N55_N44"];
N47 [color="black", label="mul_N61_N47"];
N46 [color="black", label="N62N83_mul_Imm_6_N62_add_N83_N46"];
N29 [color="black", label="N80N84_mul_N80_add_N84_N29"];
N28 [color="black", label="N86N78_mul_N86_add_N78_N28"];
N59 [color="black", label="N40N67_mul_N40_add_N67_N59"];
N23 [color="black", label="load_Imm_3_N3_N23"];
N22 [color="black", label="load_Imm_6_N2_N22"];
N21 [color="black", label="load_Imm_7_N1_N21"];
N20 [color="black", label="load_Imm_4_N9_N20"];
N27 [color="black", label="load_Imm_6_N7_N27"];
N26 [color="black", label="load_Imm_5_N6_N26"];
N25 [color="black", label="load_Imm_3_N5_N25"];
N24 [color="black", label="load_Imm_4_N4_N24"];
N67 [color="black", label="mul_Imm_11_N56_N67"];
N81 [color="black", label="store_Imm_6_N108_N81"];
N75 [color="black", label="store_Imm_0_N105_N75"];
N66 [color="black", label="N59N69_mul_Imm_6_N59_add_N69_N66"];
N76 [color="black", label="store_Imm_4_N104_N76"];
N8 [color="black", label="load_Imm_1_N24_N8"];
N9 [color="black", label="load_Imm_2_N12_N9"];
N77 [color="black", label="store_Imm_8_N103_N77"];
N71 [color="black", label="N50N57_mul_N50_add_N57_N71"];
N1 [color="black", label="load_Imm_0_N23_N1"];
N2 [color="black", label="load_Imm_3_N22_N2"];
N3 [color="black", label="load_Imm_0_N21_N3"];
N4 [color="black", label="load_Imm_7_N20_N4"];
N5 [color="black", label="load_Imm_1_N27_N5"];
N6 [color="black", label="load_Imm_8_N26_N6"];
N7 [color="black", label="load_Imm_2_N25_N7"];
N12 [color="black", label="load_Imm_8_N11_N12"];
N13 [color="black", label="load_Imm_2_N16_N13"];
N10 [color="black", label="load_Imm_6_N13_N10"];
N11 [color="black", label="load_Imm_5_N10_N11"];
N16 [color="black", label="load_Imm_5_N15_N16"];
N17 [color="black", label="load_Imm_1_N18_N17"];
N14 [color="black", label="load_Imm_4_N17_N14"];
N15 [color="black", label="load_Imm_7_N14_N15"];
N56 [color="black", label="N49N29_mul_Imm_6_N49_add_N29_N56"];
N57 [color="black", label="N48N28_mul_N48_add_N28_N57"];
N18 [color="black", label="load_Imm_0_N19_N18"];
N19 [color="black", label="load_Imm_8_N8_N19"];
N52 [color="black", label="N70N30_mul_N70_add_N30_N52"];
N53 [color="black", label="mul_N71_N53"];
N50 [color="black", label="mul_N76_N50"];
N51 [color="black", label="N77N97_mul_N77_add_N97_N51"];
N38 [color="black", label="N99N72_mul_Imm_6_N99_add_N72_N38"];
N39 [color="black", label="N92N64_mul_Imm_6_N92_add_N64_N39"];
N72 [color="black", label="mul_N51_N72"];
N30 [color="black", label="N88N90_mul_N88_add_N90_N30"];
N31 [color="black", label="N38N34_mul_N38_add_N34_N31"];
N32 [color="black", label="N39N32_mul_N39_add_N32_N32"];
N33 [color="black", label="N31N82_mul_N31_add_N82_N33"];
N34 [color="black", label="mul_N33_N34"];
N35 [color="black", label="N35N87_mul_Imm_6_N35_add_N87_N35"];
N36 [color="black", label="N36N85_mul_N36_add_N85_N36"];
N37 [color="black", label="mul_N37_N37"];
N55 -> N81;
N70 -> N57;
N54 -> N46;
N69 -> N44;
N58 -> N66;
N68 -> N56;
N49 -> N78;
N48 -> N43;
N65 -> N64;
N64 -> N60;
N63 -> N55;
N62 -> N39;
N61 -> N59;
N60 -> N39;
N41 -> N38;
N40 -> N66;
N43 -> N75;
N42 -> N49;
N45 -> N46;
N44 -> N71;
N47 -> N33;
N46 -> N80;
N29 -> N36;
N28 -> N30;
N59 -> N40;
N23 -> N59;
N23 -> N31;
N23 -> N72;
N22 -> N57;
N22 -> N51;
N22 -> N71;
N21 -> N67;
N20 -> N61;
N20 -> N37;
N20 -> N70;
N27 -> N63;
N26 -> N40;
N26 -> N57;
N26 -> N32;
N25 -> N70;
N25 -> N52;
N25 -> N44;
N24 -> N41;
N67 -> N35;
N66 -> N74;
N8 -> N28;
N8 -> N29;
N8 -> N31;
N9 -> N65;
N9 -> N59;
N9 -> N33;
N71 -> N43;
N1 -> N72;
N1 -> N69;
N1 -> N53;
N2 -> N68;
N3 -> N48;
N4 -> N32;
N4 -> N30;
N4 -> N36;
N5 -> N42;
N6 -> N45;
N6 -> N40;
N6 -> N60;
N7 -> N45;
N7 -> N71;
N7 -> N30;
N12 -> N62;
N13 -> N54;
N10 -> N65;
N10 -> N29;
N10 -> N53;
N11 -> N58;
N16 -> N64;
N16 -> N61;
N16 -> N47;
N17 -> N44;
N17 -> N50;
N17 -> N47;
N14 -> N34;
N14 -> N50;
N14 -> N37;
N15 -> N34;
N15 -> N64;
N15 -> N52;
N56 -> N73;
N57 -> N56;
N18 -> N28;
N18 -> N33;
N18 -> N69;
N19 -> N51;
N19 -> N36;
N19 -> N60;
N52 -> N51;
N53 -> N52;
N50 -> N28;
N51 -> N55;
N38 -> N76;
N39 -> N77;
N72 -> N70;
N30 -> N49;
N31 -> N32;
N32 -> N38;
N33 -> N45;
N34 -> N29;
N35 -> N79;
N36 -> N35;
N37 -> N31;
}
