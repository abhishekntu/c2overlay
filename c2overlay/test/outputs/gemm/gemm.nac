procedure foo()
{
  localvar s32 D_2410;
  localvar s32 D_2411;
  localvar s32 D_2412;
  localvar s32 D_2413;
  localvar s32 D_2414;
  localvar s32 D_2415;
  localvar s32 D_2416;
  localvar s32 D_2417;
  localvar s32 D_2418;
  localvar s32 D_2419;
  localvar s32 D_2420;
  localvar s32 D_2421;
  localvar s32 D_2422;
  localvar s32 D_2423;
  localvar s32 D_2424;
  localvar s32 D_2425;
  localvar s32 D_2426;
  localvar s32 D_2427;
  localvar s32 D_2428;
  localvar s32 D_2429;
  localvar s32 D_2430;
  localvar s32 D_2431;
  localvar s32 D_2432;
  localvar s32 D_2433;
  localvar s32 D_2434;
  localvar s32 D_2435;
  localvar s32 D_2436;
  localvar s32 D_2437;
  localvar s32 D_2438;
  localvar s32 D_2439;
  localvar s32 D_2440;
  localvar s32 D_2441;
  localvar s32 D_2442;
  localvar s32 D_2443;
  localvar s32 D_2444;
  localvar s32 D_2445;
  localvar s32 D_2446;
  localvar s32 D_2447;
  localvar s32 D_2448;
  localvar s32 D_2449;
  localvar s32 D_2450;
  localvar s32 D_2451;
  localvar s32 D_2452;
  localvar s32 D_2453;
  localvar s32 D_2454;
  localvar s32 D_2455;
  localvar s32 D_2456;
  localvar s32 D_2457;
  localvar s32 D_2458;
  localvar s32 D_2459;
  localvar s32 D_2460;
  localvar s32 D_2461;
  localvar s32 D_2462;
  localvar s32 D_2463;
  localvar s32 D_2464;
  localvar s32 D_2465;
  localvar s32 D_2466;
  localvar s32 D_2467;
  localvar s32 D_2468;
  localvar s32 D_2469;
  localvar s32 D_2470;
  localvar s32 D_2471;
  localvar s32 D_2472;
  localvar s32 D_2473;
  localvar s32 D_2474;
  localvar s32 D_2475;
  localvar s32 D_2476;
  localvar s32 D_2477;
  localvar s32 D_2478;
  localvar s32 D_2479;
  localvar s32 D_2480;
  localvar s32 D_2481;
  localvar s32 D_2482;
  localvar s32 D_2483;
  localvar s32 D_2484;
  localvar s32 D_2485;
  localvar s32 D_2486;
  localvar s32 D_2487;
  localvar s32 D_2488;
  localvar s32 D_2489;
  localvar s32 D_2490;
  localvar s32 D_2491;
  localvar s32 D_2492;
  localvar s32 D_2493;
  localvar s32 D_2494;
  localvar s32 D_2495;
  localvar s32 D_2496;
  localvar s32 D_2497;
  localvar s32 D_2498;
  localvar s32 D_2499;
  localvar s32 D_2500;
  localvar s32 D_2501;
  localvar s32 D_2502;
  localvar s32 D_2503;
  localvar s32 D_2504;
  localvar s32 D_2505;
  localvar s32 D_2506;
  localvar s32 D_2507;
  localvar s32 D_2508;
  localvar s32 a[9];
  localvar s32 b[9];
  localvar s32 c[9];
  localvar s32 tmp[9];
  localvar s32 alpha;
  localvar s32 beta;
  localvar s32 i;
  localvar s32 j;
  localvar s32 t00000;
  localvar s32 t00001;
  localvar s32 t00002;
  localvar s32 t00003;
  localvar s32 t00004;
  localvar s32 t00005;
  localvar s32 t00006;
  localvar s32 t00007;
  localvar s32 t00008;
  localvar s32 t00009;
  localvar s32 t00010;
  localvar s32 t00011;
  localvar s32 t00012;
  localvar s32 t00013;
  localvar s32 t00014;
  localvar s32 t00015;
  localvar s32 t00016;
  localvar s32 t00017;
  localvar s32 t00018;
  localvar s32 t00019;
  localvar s32 t00020;
  localvar s32 t00021;
  localvar s32 t00022;
  localvar s32 t00023;
  localvar s32 t00024;
  localvar s32 t00025;
  localvar s32 t00026;
  localvar s32 t00027;
  localvar s32 t00028;
  localvar s32 t00029;
  localvar s32 t00030;
  localvar s32 t00031;
  localvar s32 t00032;
  localvar s32 t00033;
  localvar s32 t00034;
  localvar s32 t00035;
  localvar s32 t00036;
  localvar s32 t00037;
  localvar s32 t00038;
  localvar s32 t00039;
  localvar s32 t00040;
  localvar s32 t00041;
  localvar s32 t00042;
  localvar s32 t00043;
  localvar s32 t00044;
  localvar s32 t00045;
  localvar s32 t00046;
  localvar s32 t00047;
  localvar s32 t00048;
  localvar s32 t00049;
  localvar s32 t00050;
  localvar s32 t00051;
  localvar s32 t00052;
  localvar s32 t00053;
  localvar s32 t00054;
  localvar s32 t00055;
  localvar s32 t00056;
  localvar s32 t00057;
  localvar s32 t00058;
  localvar s32 t00059;
  localvar s32 t00060;
  localvar s32 t00061;
  localvar s32 t00062;
  localvar s32 t00063;
  localvar s32 t00064;
  localvar s32 t00065;
  localvar s32 t00066;
  localvar s32 t00067;
  localvar s32 t00068;
  localvar s32 t00069;
  localvar s32 t00070;
  localvar s32 t00071;
L0005:
  alpha <= ldc 6;
  beta <= ldc 11;
  D_2410 <= load tmp,0;
  D_2411 <= mul D_2410,beta;
  D_2412 <= load a,0;
  D_2413 <= load b,0;
  D_2414 <= mul D_2412,D_2413;
  D_2415 <= load a,1;
  D_2416 <= load b,3;
  D_2417 <= mul D_2415,D_2416;
  D_2418 <= add D_2414,D_2417;
  D_2419 <= load a,2;
  D_2420 <= load b,6;
  D_2421 <= mul D_2419,D_2420;
  D_2422 <= add D_2418,D_2421;
  D_2423 <= mul D_2422,alpha;
  D_2424 <= add D_2411,D_2423;
  c <= store D_2424,0;
  D_2425 <= load tmp,1;
  D_2426 <= mul D_2425,beta;
  D_2412 <= load a,0;
  D_2427 <= load b,1;
  D_2428 <= mul D_2412,D_2427;
  D_2415 <= load a,1;
  D_2429 <= load b,4;
  D_2430 <= mul D_2415,D_2429;
  D_2431 <= add D_2428,D_2430;
  D_2419 <= load a,2;
  D_2432 <= load b,7;
  D_2433 <= mul D_2419,D_2432;
  D_2434 <= add D_2431,D_2433;
  D_2435 <= mul D_2434,alpha;
  D_2436 <= add D_2426,D_2435;
  c <= store D_2436,1;
  D_2437 <= load tmp,2;
  D_2438 <= mul D_2437,beta;
  D_2412 <= load a,0;
  D_2439 <= load b,2;
  D_2440 <= mul D_2412,D_2439;
  D_2415 <= load a,1;
  D_2441 <= load b,5;
  D_2442 <= mul D_2415,D_2441;
  D_2443 <= add D_2440,D_2442;
  D_2419 <= load a,2;
  D_2444 <= load b,8;
  D_2445 <= mul D_2419,D_2444;
  D_2446 <= add D_2443,D_2445;
  D_2447 <= mul D_2446,alpha;
  D_2448 <= add D_2438,D_2447;
  c <= store D_2448,2;
  D_2449 <= load tmp,3;
  D_2450 <= mul D_2449,beta;
  D_2451 <= load a,3;
  D_2413 <= load b,0;
  D_2452 <= mul D_2451,D_2413;
  D_2453 <= load a,4;
  D_2416 <= load b,3;
  D_2454 <= mul D_2453,D_2416;
  D_2455 <= add D_2452,D_2454;
  D_2456 <= load a,5;
  D_2420 <= load b,6;
  D_2457 <= mul D_2456,D_2420;
  D_2458 <= add D_2455,D_2457;
  D_2459 <= mul D_2458,alpha;
  D_2460 <= add D_2450,D_2459;
  c <= store D_2460,3;
  D_2461 <= load tmp,4;
  D_2462 <= mul D_2461,beta;
  D_2451 <= load a,3;
  D_2427 <= load b,1;
  D_2463 <= mul D_2451,D_2427;
  D_2453 <= load a,4;
  D_2429 <= load b,4;
  D_2464 <= mul D_2453,D_2429;
  D_2465 <= add D_2463,D_2464;
  D_2456 <= load a,5;
  D_2432 <= load b,7;
  D_2466 <= mul D_2456,D_2432;
  D_2467 <= add D_2465,D_2466;
  D_2468 <= mul D_2467,alpha;
  D_2469 <= add D_2462,D_2468;
  c <= store D_2469,4;
  D_2470 <= load tmp,5;
  D_2471 <= mul D_2470,beta;
  D_2451 <= load a,3;
  D_2439 <= load b,2;
  D_2472 <= mul D_2451,D_2439;
  D_2453 <= load a,4;
  D_2441 <= load b,5;
  D_2473 <= mul D_2453,D_2441;
  D_2474 <= add D_2472,D_2473;
  D_2456 <= load a,5;
  D_2444 <= load b,8;
  D_2475 <= mul D_2456,D_2444;
  D_2476 <= add D_2474,D_2475;
  D_2477 <= mul D_2476,alpha;
  D_2478 <= add D_2471,D_2477;
  c <= store D_2478,5;
  D_2479 <= load tmp,6;
  D_2480 <= mul D_2479,beta;
  D_2481 <= load a,6;
  D_2413 <= load b,0;
  D_2482 <= mul D_2481,D_2413;
  D_2483 <= load a,7;
  D_2416 <= load b,3;
  D_2484 <= mul D_2483,D_2416;
  D_2485 <= add D_2482,D_2484;
  D_2486 <= load a,8;
  D_2420 <= load b,6;
  D_2487 <= mul D_2486,D_2420;
  D_2488 <= add D_2485,D_2487;
  D_2489 <= mul D_2488,alpha;
  D_2490 <= add D_2480,D_2489;
  c <= store D_2490,6;
  D_2491 <= load tmp,7;
  D_2492 <= mul D_2491,beta;
  D_2481 <= load a,6;
  D_2427 <= load b,1;
  D_2493 <= mul D_2481,D_2427;
  D_2483 <= load a,7;
  D_2429 <= load b,4;
  D_2494 <= mul D_2483,D_2429;
  D_2495 <= add D_2493,D_2494;
  D_2486 <= load a,8;
  D_2432 <= load b,7;
  D_2496 <= mul D_2486,D_2432;
  D_2497 <= add D_2495,D_2496;
  D_2498 <= mul D_2497,alpha;
  D_2499 <= add D_2492,D_2498;
  c <= store D_2499,7;
  D_2500 <= load tmp,8;
  D_2501 <= mul D_2500,beta;
  D_2481 <= load a,6;
  D_2439 <= load b,2;
  D_2502 <= mul D_2481,D_2439;
  D_2483 <= load a,7;
  D_2441 <= load b,5;
  D_2503 <= mul D_2483,D_2441;
  D_2504 <= add D_2502,D_2503;
  D_2486 <= load a,8;
  D_2444 <= load b,8;
  D_2505 <= mul D_2486,D_2444;
  D_2506 <= add D_2504,D_2505;
  D_2507 <= mul D_2506,alpha;
  D_2508 <= add D_2501,D_2507;
  c <= store D_2508,8;
  L_EXIT <= jmpun;
L_EXIT:
  nop;
}
