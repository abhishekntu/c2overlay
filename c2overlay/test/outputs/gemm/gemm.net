.input N23
pinlist: N23
    
.input N22
pinlist: N22
    
.input N21
pinlist: N21
    
.input N20
pinlist: N20
    
.input N27
pinlist: N27
    
.input N26
pinlist: N26
    
.input N25
pinlist: N25
    
.input N24
pinlist: N24
    
.input N8
pinlist: N8
    
.input N9
pinlist: N9
    
.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.input N3
pinlist: N3
    
.input N4
pinlist: N4
    
.input N5
pinlist: N5
    
.input N6
pinlist: N6
    
.input N7
pinlist: N7
    
.input N12
pinlist: N12
    
.input N13
pinlist: N13
    
.input N10
pinlist: N10
    
.input N11
pinlist: N11
    
.input N16
pinlist: N16
    
.input N17
pinlist: N17
    
.input N14
pinlist: N14
    
.input N15
pinlist: N15
    
.input N18
pinlist: N18
    
.input N19
pinlist: N19
    
.output out:N74
pinlist: N74
    
.output out:N78
pinlist: N78
    
.output out:N73
pinlist: N73
    
.output out:N79
pinlist: N79
    
.output out:N80
pinlist: N80
    
.output out:N81
pinlist: N81
    
.output out:N75
pinlist: N75
    
.output out:N76
pinlist: N76
    
.output out:N77
pinlist: N77
    
.clb N55_blk
pinlist: N51 N63 open open N81 open open open open 
subblock: N55_blk 0 1 open open 4 open open open open

.clb N70_blk
pinlist: N20 N25 N72 open N70 open open open open 
subblock: N70_blk 0 1 2 open 4 open open open open
    
.clb N54_blk
pinlist: N13 open open open N54 open open open open 
subblock: N54_blk 0 open open open 4 open open open open

.clb N69_blk
pinlist: N1 N18 open open N69 open open open open 
subblock: N69_blk 0 1 open open 4 open open open open

.clb N58_blk
pinlist: N11 open open open N58 open open open open 
subblock: N58_blk 0 open open open 4 open open open open

.clb N68_blk
pinlist: N2 open open open N68 open open open open 
subblock: N68_blk 0 open open open 4 open open open open

.clb N49_blk
pinlist: N30 N42 open open N78 open open open open 
subblock: N49_blk 0 1 open open 4 open open open open

.clb N48_blk
pinlist: N3 open open open N48 open open open open 
subblock: N48_blk 0 open open open 4 open open open open

.clb N65_blk
pinlist: N9 N10 open open N65 open open open open 
subblock: N65_blk 0 1 open open 4 open open open open

.clb N64_blk
pinlist: N15 N16 N65 open N64 open open open open 
subblock: N64_blk 0 1 2 open 4 open open open open
    
.clb N63_blk
pinlist: N27 open open open N63 open open open open 
subblock: N63_blk 0 open open open 4 open open open open

.clb N62_blk
pinlist: N12 open open open N62 open open open open 
subblock: N62_blk 0 open open open 4 open open open open

.clb N61_blk
pinlist: N16 N20 open open N61 open open open open 
subblock: N61_blk 0 1 open open 4 open open open open

.clb N60_blk
pinlist: N6 N19 N64 open N60 open open open open 
subblock: N60_blk 0 1 2 open 4 open open open open
    
.clb N41_blk
pinlist: N24 open open open N41 open open open open 
subblock: N41_blk 0 open open open 4 open open open open

.clb N40_blk
pinlist: N6 N26 N59 open N40 open open open open 
subblock: N40_blk 0 1 2 open 4 open open open open
    
.clb N43_blk
pinlist: N48 N71 open open N75 open open open open 
subblock: N43_blk 0 1 open open 4 open open open open

.clb N42_blk
pinlist: N5 open open open N42 open open open open 
subblock: N42_blk 0 open open open 4 open open open open

.clb N45_blk
pinlist: N6 N7 N33 open N45 open open open open 
subblock: N45_blk 0 1 2 open 4 open open open open
    
.clb N44_blk
pinlist: N17 N25 N69 open N44 open open open open 
subblock: N44_blk 0 1 2 open 4 open open open open
    
.clb N47_blk
pinlist: N16 N17 open open N47 open open open open 
subblock: N47_blk 0 1 open open 4 open open open open

.clb N46_blk
pinlist: N45 N54 open open N80 open open open open 
subblock: N46_blk 0 1 open open 4 open open open open

.clb N29_blk
pinlist: N8 N34 N10 open N29 open open open open 
subblock: N29_blk 0 1 2 open 4 open open open open
    
.clb N28_blk
pinlist: N8 N18 N50 open N28 open open open open 
subblock: N28_blk 0 1 2 open 4 open open open open
    
.clb N59_blk
pinlist: N9 N23 N61 open N59 open open open open 
subblock: N59_blk 0 1 2 open 4 open open open open
    
.clb N67_blk
pinlist: N21 open open open N67 open open open open 
subblock: N67_blk 0 open open open 4 open open open open

.clb N66_blk
pinlist: N40 N58 open open N74 open open open open 
subblock: N66_blk 0 1 open open 4 open open open open

.clb N71_blk
pinlist: N7 N44 N22 open N71 open open open open 
subblock: N71_blk 0 1 2 open 4 open open open open
    
.clb N56_blk
pinlist: N57 N68 open open N73 open open open open 
subblock: N56_blk 0 1 open open 4 open open open open

.clb N57_blk
pinlist: N22 N26 N70 open N57 open open open open 
subblock: N57_blk 0 1 2 open 4 open open open open
    
.clb N52_blk
pinlist: N15 N25 N53 open N52 open open open open 
subblock: N52_blk 0 1 2 open 4 open open open open
    
.clb N53_blk
pinlist: N1 N10 open open N53 open open open open 
subblock: N53_blk 0 1 open open 4 open open open open

.clb N50_blk
pinlist: N14 N17 open open N50 open open open open 
subblock: N50_blk 0 1 open open 4 open open open open

.clb N51_blk
pinlist: N19 N22 N52 open N51 open open open open 
subblock: N51_blk 0 1 2 open 4 open open open open
    
.clb N38_blk
pinlist: N32 N41 open open N76 open open open open 
subblock: N38_blk 0 1 open open 4 open open open open

.clb N39_blk
pinlist: N60 N62 open open N77 open open open open 
subblock: N39_blk 0 1 open open 4 open open open open

.clb N72_blk
pinlist: N1 N23 open open N72 open open open open 
subblock: N72_blk 0 1 open open 4 open open open open

.clb N30_blk
pinlist: N4 N7 N28 open N30 open open open open 
subblock: N30_blk 0 1 2 open 4 open open open open
    
.clb N31_blk
pinlist: N8 N37 N23 open N31 open open open open 
subblock: N31_blk 0 1 2 open 4 open open open open
    
.clb N32_blk
pinlist: N4 N31 N26 open N32 open open open open 
subblock: N32_blk 0 1 2 open 4 open open open open
    
.clb N33_blk
pinlist: N9 N18 N47 open N33 open open open open 
subblock: N33_blk 0 1 2 open 4 open open open open
    
.clb N34_blk
pinlist: N14 N15 open open N34 open open open open 
subblock: N34_blk 0 1 open open 4 open open open open

.clb N35_blk
pinlist: N36 N67 open open N79 open open open open 
subblock: N35_blk 0 1 open open 4 open open open open

.clb N36_blk
pinlist: N4 N29 N19 open N36 open open open open 
subblock: N36_blk 0 1 2 open 4 open open open open
    
.clb N37_blk
pinlist: N14 N20 open open N37 open open open open 
subblock: N37_blk 0 1 open open 4 open open open open

