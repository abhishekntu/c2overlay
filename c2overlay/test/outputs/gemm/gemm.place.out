Netlist file: test.net   Architecture file: test.arch.xml
Array size: 9 x 9 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N23		3	0	0	#0
N22		10	4	0	#1
N21		2	10	0	#2
N20		6	0	0	#3
N27		10	8	0	#4
N26		2	0	0	#5
N25		10	6	0	#6
N24		0	7	0	#7
N8		0	6	0	#8
N9		4	0	0	#9
N1		10	5	0	#10
N2		10	1	0	#11
N3		9	10	0	#12
N4		0	4	0	#13
N5		1	10	0	#14
N6		3	10	0	#15
N7		0	5	0	#16
N12		9	0	0	#17
N13		10	7	0	#18
N10		0	3	0	#19
N11		1	0	0	#20
N16		5	0	0	#21
N17		7	10	0	#22
N14		4	10	0	#23
N15		10	3	0	#24
N18		5	10	0	#25
N19		8	0	0	#26
out:N74		0	1	0	#27
out:N78		0	8	0	#28
out:N73		10	2	0	#29
out:N79		0	9	0	#30
out:N80		6	10	0	#31
out:N81		10	9	0	#32
out:N75		8	10	0	#33
out:N76		0	2	0	#34
out:N77		7	0	0	#35
N55_blk		9	3	0	#36
N70_blk		7	2	0	#37
N54_blk		7	5	0	#38
N69_blk		6	4	0	#39
N58_blk		1	1	0	#40
N68_blk		9	1	0	#41
N49_blk		1	4	0	#42
N48_blk		9	5	0	#43
N65_blk		5	3	0	#44
N64_blk		5	2	0	#45
N63_blk		9	4	0	#46
N62_blk		8	1	0	#47
N61_blk		5	1	0	#48
N60_blk		6	1	0	#49
N41_blk		1	3	0	#50
N40_blk		3	1	0	#51
N43_blk		8	5	0	#52
N42_blk		1	5	0	#53
N45_blk		5	5	0	#54
N44_blk		7	4	0	#55
N47_blk		5	4	0	#56
N46_blk		6	5	0	#57
N29_blk		3	3	0	#58
N28_blk		3	4	0	#59
N59_blk		4	1	0	#60
N67_blk		3	5	0	#61
N66_blk		2	1	0	#62
N71_blk		8	4	0	#63
N56_blk		9	2	0	#64
N57_blk		8	2	0	#65
N52_blk		7	3	0	#66
N53_blk		6	3	0	#67
N50_blk		4	5	0	#68
N51_blk		8	3	0	#69
N38_blk		1	2	0	#70
N39_blk		7	1	0	#71
N72_blk		6	2	0	#72
N30_blk		2	4	0	#73
N31_blk		3	2	0	#74
N32_blk		2	2	0	#75
N33_blk		4	4	0	#76
N34_blk		4	3	0	#77
N35_blk		2	5	0	#78
N36_blk		2	3	0	#79
N37_blk		4	2	0	#80
